README.md

```diff
@@ -74,6 +74,49 @@ You can run individual actions like this:
 v my-project app bash
 ```
 
+### Adding more advanced actions
+You can always use the feature set of the `sh` shell. Like operators, conditions and variables. So you can build powerful workflows which saves you brain capacity and time.
+```yaml
+services:
+  app:
+    actions:
+      # Shell Operators
+      #
+      # use yaml multiline strings and shell operators to run multiple
+      # commands after the other.
+      start: >
+        docker-compose up -d &&
+        echo "startup done" &&
+        docker-compose exec app npm run watch &&
+        code .
+
+      # Shell Variables
+      #
+      # $@ is a shell variable which holds all passed arguments
+      # so it is possible to pass all arguments to a command by running:
+      # v my-project app exec npm install double-vue
+      exec: docker-compose exec app $@
+      # You can pass individual shell arguments as well by using $1..$[x]
+      create-branch: git checkout -b feature/$1
+
+      # Custom Shell
+      #
+      # If running your commands in a simple `sh` is not enough you could
+      # launch your favorite shell as well to leverage more complex tasks
+      watch: zsh -c "source ~/.zshrc && nvm use && npm run watch"
+
+```
+
+## Terminology
+
+| Term | Description | Example |
+| ---- | ----------- | ------- |
+| Project | Combines logically and technical related services | TodoList |
```

Comment:
The "logically and technically" sounds a bit clunky to me. I understand where you were going with it, and I thought about how to incorporate it, but for me a simpler description already does the job:
```suggestion
| Project | Groups services that are related to each other | TodoList |
```